### DISCUSSION TITLE
How to get the current user ID/user object?

### DISCUSSION THREAD

**Discussion by bmarshall511** [wpengine/faustjs#944] [Q&A]
New to Faust.js and trying to get the current user ID/object after a user has logged in via the `login` function in the `useLogin` hook. Took a look at the docs & data that those functions return, but couldn't find anything.

**Comment by josephfusco** [comment]
Hey @bmarshall511, I believe you are correct with it not being available. We are actively developing the next major version of faust (currently known as faust-nx) and will discuss with the team.

In the meantime, you should be able to query WP directly for that information https://www.wpgraphql.com/docs/users/#querying-users
